### PR TITLE
Sublime depends on gtk2

### PR DESCRIPTION
**Pull request by helge000**
This was mentioned in the FAQ, however, gtk2 is not installed.

### CODE REVIEW

**Comment by JL2210** [comment]
It's not an issue with GTK2, it's just misinformation in the FAQ. Follow
the instructions, just change instances of 2.0 to 3.0.

**Comment by helge000** [comment]
@JL2210, to clarify: We do not need gtk2 for sublime_text to work? Or should we use rather depend on gtk3?

I accidentally wiped my chromebook two times in a row and both times sublime would only work after gtk2 was installed.

I only checked primary deps for [sommelier](https://github.com/skycocker/chromebrew/blob/master/packages/sommelier.rb#L23), it does not seem  to pull in any of the gtk packages.

### Ref
From [FAQ](https://github.com/skycocker/chromebrew/wiki/FAQ): 
> Q: I'm having troubles with font size in apps depending on GTK2. What do I do?
> A: Packages like gimp and sublime_text still depend on GTK2.  [...]

**Comment by JL2210** [comment]
If that's the case, then this is OK. @uberhacker?

File: packages/sublime_text.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Sublime_text < Package
   })
 
   depends_on 'sommelier'
+  depends_on 'gtk2'
```
Comment (uberhacker) [inline packages/sublime_text.rb]: Can you please move this above `depends_on 'sommelier'`?  The reason is there are post install instructions that would be missed.

Comment (helge000) [reply packages/sublime_text.rb]: Done

**Pull request merged**